lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -57,6 +86,8 @@ defmodule Mix.Tasks.Xref do
         unreachable()
       ["callers", callee] ->
         callers(callee)
+      ["graph"] ->
+        graph(opts)
       _ ->
         Mix.raise "xref expects one of the following commands: warnings, unreachable, callers CALLEE"
```

Comment:
Maybe easier to say "xref doesn't support this command, see mix help xref for more information"?